app/code/Codifi/Sales/Setup/Patch/Data/AddOrderTypeToCoreConfigDataWithLabel.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Codifi_Sales
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\Sales\Setup\Patch\Data;
+
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\App\Config\Storage\WriterInterface;
+use Magento\Framework\Serialize\SerializerInterface;
+use Codifi\Sales\Setup\Patch\Data\AddOrderTypeToCoreConfigData;
+
+/**
+ * Class AddOrderTypeToCoreConfigDataWithLabel
+ * @package Codifi\Sales\Setup\Patch\Data
+ */
+class AddOrderTypeToCoreConfigDataWithLabel implements DataPatchInterface
+{
+    /**
+     * Serializer interface
+     *
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * Writer Interface
+     *
+     * @var WriterInterface
+     */
+    private $configWriter;
+
+    /**
+     * Path to config
+     */
+    const ORDER_TYPE_XML_PATH = 'codifi_sales/order/order_type';
```

Comment:
1. Constant the from previous data patch can be used
2. Constants are declared before properties